Library/Formula/v8.rb

```diff
@@ -21,6 +21,7 @@ class V8 < Formula
 
   depends_on :python => :build # gyp doesn't run under 2.6 or lower
   depends_on "readline" => :optional
+  depends_on "icu4c" => :recommended
```

Comment:
Yes, it looks like it shipped in default distribution. At least this true in clean vagrant boxes.
```
vagrant@debian-jessie:~$ aptitude search '~i icu'
i   libicu52                                   - International Components for Unicode                 
```

and

```
vagrant@vagrant-ubuntu-wily-64:~$ aptitude search '~i icu'
i   libicu55                                   - International Components for Unicode                 
```